zeus/views/site.py

```diff
@@ -92,7 +92,9 @@ class F(STVBallotForm):
         if not os.path.exists(filename):
             return HttpResponseRedirect(reverse('stv_count') + "?reset=1")
 
-        wrapper = FileWrapper(open(filename))
+        with open(filename) as f:
+            wrapper = FileWrapper(f)
```

Comment:
This is incorrect. You are creating an object with an open file, but then you immediately close the file, making the object unusable.

In addition, it appears that there is now a standard way of serving files via Django: a [FileResponse](https://docs.djangoproject.com/en/1.11/ref/request-response/#fileresponse-objects). So you probably should use that instead of the `FileWrapper`.